lib/google_contacts_api/groups.rb

```diff
@@ -23,5 +23,28 @@ def get_groups(params = {})
       end
       GoogleContactsApi::GroupSet.new(response.body, @api)
     end
+
+    def create_group(title)
+      response = @api.post('groups/default/full', create_group_xml(title), {},
+                          'Content-Type' => 'application/atom+xml')
+      Group.new(Hashie::Mash.new(JSON.parse(response.body)).entry)
+    end
+
+    def delete_group(group)
```

Comment:
Yeah, implementing it on the group does make sense and just letting it take an id would make it more generally usable. Perhaps I can move (or alias) it next time I work on the gem.